hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@pkdash Once publisher element is created, we don't allow it to be updated since published resources are not editable. I suggest just to raise ValidationError in update() indicating publisher element cannot be updated without doing any additional checking.